recipe/meta.yaml

```diff
@@ -24,22 +25,23 @@ requirements:
     - setuptools_scm >=3.4.3
     - wheel
   run:
+    - python
     - autopep8 >=1.6.0,<1.7.0
-    - flake8 >=4.0.0,<4.1.0
+    - docstring-to-markdown
+    - flake8 >=5.0.0,<7.0.0
     - jedi >=0.17.2,<0.19.0
-    - mccabe >=0.6.0,<0.8.0
+    - mccabe >=0.7.0,<0.8.0
     - pluggy >=1.0.0
     - python-lsp-jsonrpc >=1.0.0
-    - pycodestyle >=2.8.0,<2.9.0
-    - pydocstyle >=2.0.0
-    - pyflakes >=2.4.0,<2.5.0
-    - pylint >=2.5.0
-    - python
-    - rope >=0.10.5
-    - setuptools >=39.0.0
+    # - pycodestyle >=2.9.0,<2.11.0
+    - pycodestyle =2.10.0
+    - pydocstyle >=6.2.0,<6.4.0
```

Comment:
`pydocstyle >=6.2.0,<6.3.0`, see https://github.com/python-lsp/python-lsp-server/blob/d28ce21e151c2de0bf47ff1183b512c96e3b97d8/pyproject.toml#L34

Reply:
Hi there! `Pydocstyle` has an updated pinning on the main developer page. [See here](https://github.com/python-lsp/python-lsp-server/blob/e3411981fb7ba5147923bbc6160353feffd659b1/pyproject.toml#L34).